Index: configure.ac
===================================================================
--- configure.ac	(revision 685281)
+++ configure.ac	(working copy)
@@ -64,7 +64,6 @@
 	# -Wunreachable-code -Wpadded -Winline
 	# -Wshadow - warns about boost headers.
 	if test "${enableval}" = yes; then
-	    gl_COMPILER_FLAGS(-Werror)
 	    gl_COMPILER_FLAGS(-pedantic)
 	    gl_COMPILER_FLAGS(-Wall)
 	    gl_COMPILER_FLAGS(-Wextra)
Index: src/Makefile.am
===================================================================
--- src/Makefile.am	(revision 685281)
+++ src/Makefile.am	(working copy)
@@ -1,6 +1,6 @@
 SUBDIRS = . tests
 
-EXTRA_DIST= $(platform_dist) $(rgen_srcs)
+EXTRA_DIST= $(platform_dist) $(rgen_srcs) $(top_srcdir)/src/boost
 
 ## Generated code
 
