Index: configure.ac
===================================================================
--- configure.ac	(revision 718931)
+++ configure.ac	(working copy)
@@ -66,7 +66,6 @@
 	# -Wunreachable-code -Wpadded -Winline
 	# -Wshadow - warns about boost headers.
 	if test "${enableval}" = yes; then
-	    gl_COMPILER_FLAGS(-Werror)
 	    gl_COMPILER_FLAGS(-pedantic)
 	    gl_COMPILER_FLAGS(-Wall)
 	    gl_COMPILER_FLAGS(-Wextra)
Index: src/Makefile.am
===================================================================
--- src/Makefile.am	(revision 718931)
+++ src/Makefile.am	(working copy)
@@ -37,7 +37,7 @@
   qpid/broker/windows/BrokerDefaults.cpp \
   qpid/broker/windows/SaslAuthenticator.cpp
 
-EXTRA_DIST= $(platform_dist) $(rgen_srcs) $(windows_dist)
+EXTRA_DIST= $(platform_dist) $(rgen_srcs) $(windows_dist) $(top_srcdir)/src/boost
 
 ## Generated code
 
