Index: configure.ac
===================================================================
--- configure.ac	(revision 685217)
+++ configure.ac	(working copy)
@@ -66,7 +66,6 @@
 # -Wshadow - warns about boost headers.
 
 if test "${enableval}" = yes; then
-  gl_COMPILER_FLAGS(-Werror)
   gl_COMPILER_FLAGS(-pedantic)
   gl_COMPILER_FLAGS(-Wall)
   gl_COMPILER_FLAGS(-Wextra)
Index: src/tests/.valgrind.supp
===================================================================
--- src/tests/.valgrind.supp	(revision 685217)
+++ src/tests/.valgrind.supp	(working copy)
@@ -70,19 +70,6 @@
    fun:pthread_create@@GLIBC_2.2.5
    fun:_ZN4qpid6broker5Timer5startEv
    fun:_ZN4qpid6broker5TimerC1Ev
-   fun:_ZN4qpid6broker10DtxManagerC1Ev
-   fun:_ZN4qpid6broker6BrokerC1ERKNS1_7OptionsE
-   fun:_ZN4qpid6broker6Broker6createERKNS1_7OptionsE
-   fun:_ZN15SessionFixtureTI15ProxyConnectionEC2Ev
-   fun:_Z14testQueueQueryv
-   fun:_ZN5boost9unit_test9ut_detail17unit_test_monitor8functionEv
-   obj:/usr/lib64/libboost_unit_test_framework.so.1.32.0
-   fun:_ZN5boost17execution_monitor7executeEbi
-   fun:_ZN5boost9unit_test9ut_detail17unit_test_monitor21execute_and_translateEPNS0_9test_caseEMS3_FvvEi
-   fun:_ZN5boost9unit_test9test_case3runEv
-   fun:_ZN5boost9unit_test10test_suite6do_runEv
-   fun:_ZN5boost9unit_test9test_case3runEv
-   fun:main
 }
 
 {
@@ -151,29 +138,6 @@
    fun:calloc
    fun:_dl_allocate_tls
    fun:pthread_create@@GLIBC_2.2.5
-   fun:_ZN4qpid6broker5Timer5startEv
-   fun:_ZN4qpid6broker5TimerC1Ev
-   fun:_ZN4qpid6broker10DtxManagerC1Ev
-   fun:_ZN4qpid6broker6BrokerC1ERKNS1_7OptionsE
-   fun:_ZN4qpid6broker6Broker6createERKNS1_7OptionsE
-   fun:_ZN20ClientSessionFixtureC1Ev
-   fun:_Z14testQueueQueryv
-   fun:_ZN5boost9unit_test9ut_detail17unit_test_monitor8functionEv
-   obj:/usr/lib64/libboost_unit_test_framework.so.1.32.0
-   fun:_ZN5boost17execution_monitor7executeEbi
-   fun:_ZN5boost9unit_test9ut_detail17unit_test_monitor21execute_and_translateEPNS0_9test_caseEMS3_FvvEi
-   fun:_ZN5boost9unit_test9test_case3runEv
-   fun:_ZN5boost9unit_test10test_suite6do_runEv
-   fun:_ZN5boost9unit_test9test_case3runEv
-   fun:main
-}
-
-{
-   INVESTIGATE
-   Memcheck:Leak
-   fun:calloc
-   fun:_dl_allocate_tls
-   fun:pthread_create@@GLIBC_2.2.5
    fun:_ZN4qpid6client9Connector4initEv
 }
 
@@ -191,4 +155,3 @@
    fun:strdup
    fun:_ZN4qpid7Options5parseEiPPcRKSsb
 }
-
Index: src/Makefile.am
===================================================================
--- src/Makefile.am	(revision 685217)
+++ src/Makefile.am	(working copy)
@@ -1,6 +1,6 @@
 SUBDIRS = . tests
 
-EXTRA_DIST= $(platform_dist) $(rgen_srcs)
+EXTRA_DIST= $(platform_dist) $(rgen_srcs) $(top_srcdir)/src/boost
 
 ## Generated code
 
